src/ORM/Filters/ExactMatchFilter.php

```diff
@@ -133,17 +140,25 @@ protected function manyFilter(DataQuery $query, $inclusive)
         }
 
         $connective = '';
+        $setValuesToEmpty = false;
         if (empty($values)) {
             $predicate = '';
         } elseif ($caseSensitive === null) {
             // For queries using the default collation (no explicit case) we can use the WHERE .. NOT IN .. syntax,
             // providing simpler SQL than many WHERE .. AND .. fragments.
             $column = $this->getDbName();
-            $placeholders = DB::placeholders($values);
+            $usePlaceholders = $this->usePlaceholders($column, $values);
+            if ($usePlaceholders) {
+                $in = DB::placeholders($values);
+            } else {
+                // explitly including space after comma so that sql wraps nicely on screen when using showqueries=1
```

Comment:
```suggestion
                // explicitly including space after comma to match the default for DB::placeholders
```
I was going to ask you to remove the space before I noticed placeholders adds it by default as well.

Reply:
Updated